fluentd/vendored_gem_src/fluentd/lib/fluent/plugin/in_tail.rb

```diff
@@ -510,6 +510,13 @@ def initialize(path, rotate_wait, pe, log, read_from_head, enable_watch_timer, e
         @from_encoding = from_encoding
         @encoding = encoding
         @open_on_every_update = open_on_every_update
+        @totalbytesread=0
+        @totalbytesavailable=0
+        @inodelastfsize_map={}
```

Comment:
Same as next comment

Reply:
@eranra @alanconway resolved this in the new implementation.